source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -289,7 +289,6 @@ namespace AdaptiveCards { namespace Rendering { namespace Uwp
     [version(NTDDI_WIN10_RS1)]
     typedef [v1_enum] enum ErrorStatusCode {
         InvalidJson = 0,
-        UnsupportedSchemaVersion,
```

Comment:
>UnsupportedSchemaVersion, [](start = 8, length = 25)

Do you have any existing clients that might be affected by this breaking change?

Reply:
Yes, and this is why we're doing this breaking change now.

Reply:
Yes, this is why we're doing this now, before we release.


---
In reply to: [172656255](https://github.com/Microsoft/AdaptiveCards/pull/1222#discussion_r172656255) [](ancestors = 172656255)